lib/media/track/mediatrack.js

```diff
@@ -71,7 +71,6 @@ class MediaTrack extends Track {
     this._log.debug('Started');
     this._isStarted = true;
     if (this._dummyEl) {
-      this._detachElement(this._dummyEl);
```

Comment:
@markandrus Since you've removed the `_detachElement()` call here, I don't see it being called anywhere in `VideoTrack` (it is being called in `AudioTrack`). Any reason why this is being removed from the common base class?